rasa/core/policies/policy.py

```diff
@@ -354,49 +344,55 @@ def _metadata_filename(cls) -> Text:
         """
         pass
 
-    def persist(self, path: Union[Text, Path]) -> None:
-        """Persists the policy to storage.
-
-        Args:
-            path: Path to persist policy to.
-        """
-        # not all policies have a featurizer
-        if self.featurizer is not None:
-            self.featurizer.persist(path)
+    def persist(self) -> None:
+        """Persists the policy to storage."""
+        with self._model_storage.write_to(self._resource) as path:
```

Comment:
💡 `GraphComponent`s can persist themselves using their `resource` locator. The `model_storage` will provide a path for them which they can write to. See `load` if you want to see how to read from a `model_storage`